doc/manual/index.rst

```diff
@@ -127,7 +126,6 @@ Additional information about various MNE-C tools.
    c_reference
    gui/analyze
    gui/browse
-   c_cookbook
```

Comment:
why have you removed this? @yousrabk had a PR coming with nicer layouts etc. Not sure how far it has progressed but it may result in rebase conflicts ...